crates/full-node/sov-stf-runner/src/http/ws.rs

```diff
@@ -82,13 +83,13 @@ async fn socket_reader_task(
             };
             match msg {
                 Message::Text(text) => {
-                    handle_rpc_message(&text, &msg_tx, &rpc_methods, false, &shutdown_receiver).await;
+                    spawn(handle_rpc_message(text, msg_tx.clone(), rpc_methods.clone(), false, shutdown_receiver.clone()));
                 }
                 Message::Binary(data) => {
                     // Parse binary frame as UTF-8 JSON-RPC request
-                    match std::str::from_utf8(&data) {
+                    match String::from_utf8(data) {
                         Ok(text) => {
-                            handle_rpc_message(text, &msg_tx, &rpc_methods, true, &shutdown_receiver).await;
+                            spawn(handle_rpc_message(text, msg_tx.clone(), rpc_methods.clone(), true, shutdown_receiver.clone()));
```

Comment:
It appears that previously, messages were handled in order, but after this change, they can be processed out of order. For example, if two messages are executed in quick succession, they may spawn simultaneously, and the second one could finish before the first. Is that ok?